bento_beacon/config_files/config.py

```diff
@@ -18,12 +18,16 @@ class Config:
         "info": "record"
     }
 
+    BENTO_DOMAIN = os.environ.get("BENTOV2_DOMAIN")
     BEACON_BASE_URL = os.environ.get("BEACON_BASE_URL")
+    BENTO_PUBLIC_URL = "https://" + BENTO_DOMAIN
```

Comment:
theres an envion variable for this which might be a better source of truth by default, with a fallback to this logic using BENTO_DOMAIN?

Reply:
was just thinking it makes sense to use the env file as the single source of truth, although yeah I don't foresee this logic changing so whatever really